textfsm_test.py

```diff
@@ -846,5 +857,27 @@ def testFillup(self):
         str(result))
 
 
+  def testRepeated(self):
+    """Repeated option should work ok."""
+    tplt = """Value Repeated repeatedKey (\S+)
+Value Repeated repeatedValue (\S+)
+Value normalData (\S+)
+Value normalData2 (\S+)
+Value Repeated repeatedUnused (\S+)
```

Comment:
List and Repeated should be mutually exclusive (and tested for this)

Reply:
I don't see why List and Repeated have to be mutually exclusive. It is possible to have two lines that require the repeated functionality and are related to each other in a way that makes sense to list them. All entries would end up being a list of lists: [[repeated_data1, repeated_data2]]. Unless you have an objection to this I will add tests to ensure this functionality.

Reply:
Sure, please add test for the case where there is a Repeated,List